### DISCUSSION TITLE
Initialization for orbit

### DISCUSSION THREAD

**Discussion by jonsberndt** [JSBSim-Team/jsbsim#704] [Ideas]
Just so I don't forget:

Would be nice to be able to initialize an object in orbit. I'd imagine that the initialization _could_ include these parameters

- altitude above reference surface at equator
- apogee (and perigee if not circular)
- inclination
- initial position. Could be latitude and ascending/descending, or true anomaly, lat/lon, or something ...

**Comment by jonsberndt** [comment]
I believe I have found a way to model this without any JSBSim mods. I hope to share the process soon.

**Reply by bcoconni** [reply]
> I hope to share the process soon.

That'd be great !
